client/packages/invoices/src/Prescriptions/LineEditView/NavBar.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * TO-DO: Make this a generic re-useable component (in "common"), and update
+ * styling to match new designs
+ */
+
+import {
```

Comment:
This component just bare-bones for now. The UI designs have new colour scheme, etc, but don't want to start down that path just yet as that requires a more global approach.